cypress/integration/plugins/anomaly-detection-dashboards-plugin/historical_analysis_spec.js

```diff
@@ -58,17 +58,17 @@ describe('Historical results page', () => {
         cy.getElementByTestId('viewSampleDetectorLink').click();
         cy.getElementByTestId('configurationsTab').click();
         cy.getElementByTestId('detectorIdCell').within(() => {
-        cy.get('.euiText--medium')
-          .invoke('text')
-          .then((detectorId) => {
-            cy.log('Stopping detector with ID: ' + detectorId);
-            cy.stopDetector(detectorId);
-            cy.wait(10000);
-            cy.log('Deleting detector with ID: ' + detectorId);
-            cy.deleteDetector(detectorId);
-            cy.log('Deleting index with name: ' + indexName);
-            cy.deleteIndex(indexName);
-          });
+          cy.get('.euiText--medium')
+            .invoke('text')
+            .then((detectorId) => {
+              cy.log('Stopping detector with ID: ' + detectorId);
+              cy.stopDetector(detectorId);
+              cy.wait(10000);
+              cy.log('Deleting detector with ID: ' + detectorId);
+              cy.deleteDetector(detectorId);
+              // cy.log('Deleting index with name: ' + indexName);
+              // cy.deleteIndex(indexName);
```

Comment:
This should not break any test since `indexName` is actually not defined anywhere. I'm not sure how the tests were running for this to begin with.

Reply:
hey @ohltyler , do you have context for this test? it's weird that indexName is not defined